content/enterprise_influxdb/v1.9/guides/migration.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Migrate InfluxDB OSS instances to InfluxDB Enterprise clusters
+description: >
+  Migrate a running instance of InfluxDB open source (OSS) to an InfluxDB Enterprise cluster.
+aliases:
+  - /enterprise/v1.9/guides/migration/
+menu:
+  enterprise_influxdb_1_9:
+    name: Migrate InfluxDB OSS to Enterprise
+    weight: 10
+    parent: Guides
+---
+
+Migrate a running instance of InfluxDB open source (OSS) to an InfluxDB Enterprise cluster.
+
+## Prerequisites
+
+- An InfluxDB OSS instance running **InfluxDB 1.7.10 or later**.
+- An InfluxDB Enterprise cluster running **InfluxDB Enterprise 1.7.10 or later**
+- Your **OSS and Enterprise version is the same**, for example, InfluxDB 1.8 and InfluxDB Enterprise 1.8.
```

Comment:
I'm unclear why An InfluxDB Enterprise cluster running 1.7.10 or later is a pre-requisite.  I believe the instructions have the user install and setup the InfluxDB Enterprise meta nodes.  However, I think these instructions are kind of confusing.  I believe we should have two separate guides.... one for upgrading OSS to Enterprise (where the Enterprise setup is "fresh") and another for adding an OSS node to an existing InfluxDB Enterprise installation (some what more rare...but there are other considerations when doing this -- like rebalancing).  

The version numbers between OSS and Enterprise will no longer be the same.  So, we can suggest that the user ensure that they are running on at least 1.8.x and preferably 1.8.latest.
InfluxDB Enterprise will be 1.9 (which is the current release).

Also, after the upgrade --- I think the rebalancing steps are potentially confusing?  different steps for a 2 data node cluster vs. a 4...   (there isn't much rebalancing to do for a 2 node cluster other than -> 1. Use the [ALTER RETENTION POLICY](/influxdb/v1.9/query_language/manage-database/#modify-retention-policies-with-alter-retention-policy) statement to increase the [replication factor](/enterprise_influxdb/v1.9/concepts/glossary/#replication-factor) on all existing retention polices to the number of data nodes in your cluster._